picoboot_connection/picoboot_connection.h

```diff
@@ -39,6 +39,7 @@ int picoboot_enter_cmd_xip(libusb_device_handle *usb_device);
 int picoboot_exit_xip(libusb_device_handle *usb_device);
 int picoboot_reboot(libusb_device_handle *usb_device, uint32_t pc, uint32_t sp, uint32_t delay_ms);
 int picoboot_exec(libusb_device_handle *usb_device, uint32_t addr);
+int picoboot_flash_range_erase(libusb_device_handle *, uint32_t, uint32_t, uint32_t, uint8_t);
```

Comment:
I just wanted to keep the lines to a reasonable length so you can have a text editor and terminal on the same screen.  It would have been reasonable to have the names and insert a line break as well.

Reply:
Isn't the header-file often used as a quick-reference list of the functions and their parameters? That's why I'd argue for including the parameter-names, but ultimately that's Graham's decision.